### PR TITLE
Adding Config for Pulpcore 3.22

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: theforeman.org/yaml/includes/pulpcore_versions.yaml.inc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - '3.16'
 - '3.17'
 - '3.18'
-- '3.21'
\ No newline at end of file
+- '3.21'
+- '3.22'
```
Comment (ekohl) [inline theforeman.org/yaml/includes/pulpcore_versions.yaml.inc:5]: Please ensure a line end is here.

Comment (Odilhao) [reply theforeman.org/yaml/includes/pulpcore_versions.yaml.inc:5]: Ack

Comment (ekohl) [reply theforeman.org/yaml/includes/pulpcore_versions.yaml.inc:5]: It still shows the symbol that there's no line ending.

Comment (Odilhao) [reply theforeman.org/yaml/includes/pulpcore_versions.yaml.inc:5]: Had to use vim, lol

Comment (ekohl) [reply theforeman.org/yaml/includes/pulpcore_versions.yaml.inc:5]: I don't know why you keep marking it as resolved, but it's still not there.

**Review by ekohl — APPROVED** [review]

**Pull request merged**